src/main/java/org/minions/devfund/reinaldo/battleship/Ocean.java

```diff
@@ -0,0 +1,166 @@
+package org.minions.devfund.reinaldo.battleship;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * Ocean class.
+ */
+public class Ocean {
+    private Ship[][] ships;
+    private int shotsFired; //int shotsFired - The total number of shots fired by the user.
+    private int hitCount; //int hitCount - The number of times a shot hit a ship.
+    private int shipsSunk; //The number of ships sunk. Remember that you have a total of 13 ships.
+
+    static final int SHIPLENGTH = 20;
+    private static final int SHIPS_QUANTITY = 13;
+
+    private static final List<String> SHIPS_TO_PLACE;
+
+    static {
+        SHIPS_TO_PLACE = new ArrayList<>();
+        SHIPS_TO_PLACE.add("BattleShip");
+        SHIPS_TO_PLACE.add("BattleCruiser");
+        SHIPS_TO_PLACE.add("Cruiser");
+        SHIPS_TO_PLACE.add("Cruiser");
+        SHIPS_TO_PLACE.add("LightCruiser");
+        SHIPS_TO_PLACE.add("LightCruiser");
+        SHIPS_TO_PLACE.add("Destroyer");
+        SHIPS_TO_PLACE.add("Destroyer");
+        SHIPS_TO_PLACE.add("Destroyer");
+        SHIPS_TO_PLACE.add("Submarine");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Submarine" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)